examples/tensorflow/common/callbacks.py

```diff
@@ -127,7 +127,10 @@ def _calculate_metrics(self) -> MutableMapping[str, Any]:
 
     def _calculate_lr(self) -> int:
         """Calculates the learning rate given the current step."""
-        return get_scalar_from_tensor(self._get_base_optimizer().lr(self.step))
+        lr = self._get_base_optimizer().lr
```

Comment:
Need to process to cases
- `tf.keras.optimizers.Adam` has a new API, where `.lr` returns current `lr` tensor
- `tfa.optimizers.SGDW` has old API, where `.lr()` is a scheduler method accepting step